src/cf-icons/usage.md

```diff
@@ -43,22 +44,23 @@ Overwrite them in your own project by duplicating the variable `@key: value`.
 
 ## The basics
 
-The cf-icon-prefix class applies all shared icon styles including the font family.
-By default, this class will be named `cf-icon` but it can be changed in the settings.
-All icons must use three classes, one for the base class, one to select the
-desired icon, and one for the chosen pseudo-element. For example:
+The cf-icon-prefix class applies all shared icon styles including the font
```

Comment:
Minor: "The cf-icon-prefix class" ➡️ "The class defined by the `@cf-icon-prefix` variable.